### ISSUE TITLE
Menu buttons and menu

### ISSUE THREAD

**Issue by zcjess** [zcjess/Ecoach-accessibility-issues#11]
<img width="672" alt="image" src="https://user-images.githubusercontent.com/90011962/225053408-e33780b4-5970-4e2c-be0c-0f5602f6014a.png">

1. The element with role button has [aria-haspopup](https://w3c.github.io/aria/#aria-haspopup) set to either menu or true.
2. When the menu is displayed, the element with role button has [aria-expanded](https://w3c.github.io/aria/#aria-expanded) set to true. When the menu is hidden, it is recommended that aria-expanded is not present. If aria-expanded is specified when the menu is hidden, it is set to false.

Keyboard interaction:
1. With focus on the button:
Enter: opens the menu and places focus on the first menu item.
Space: Opens the menu and places focus on the first menu item.

<img width="260" alt="image" src="https://user-images.githubusercontent.com/90011962/225053368-edb39b1a-3681-42b0-bf13-e895a88ec198.png">

1. A parent menuitem has [aria-haspopup](https://w3c.github.io/aria/#aria-haspopup) set to either menu or true.
2. A parent menuitem has [aria-expanded](https://w3c.github.io/aria/#aria-expanded) set to false when its child menu is not visible and set to true when the child menu is visible.
3. An element with role menu either has:
1）[aria-labelledby](https://w3c.github.io/aria/#aria-labelledby) set to a value that refers to the menuitem or button that controls its display.
2）A label provided by [aria-label](https://w3c.github.io/aria/#aria-label).

Keyboard:
1. down arrow: When focus is in a menu, moves focus to the next item, optionally wrapping from the last to the first.
2. up arrow: When focus is in a menu, moves focus to the previous item, optionally wrapping from the first to the last.
3. right arrow: When focus is in a menubar, moves focus to the next item, optionally wrapping from the last to the first.
4. left arrow: When focus is in a menubar, moves focus to the previous item, optionally wrapping from the first to the last.
